prow/cmd/deck/main.go

```diff
@@ -116,6 +121,36 @@ func handleData(ja *JobAgent) http.HandlerFunc {
 
 type logClient interface {
 	GetJobLog(job, id string) ([]byte, error)
+	GetJobLogStream(job, id string, options map[string]string) (io.ReadCloser, error)
```

Comment:
I agree that it is a design choice, I will wait for opinions. I like to have the ability to have all options enabled.